apps/scout/client.py

```
@@ -0,0 +1,63 @@
+        self.api_key = api_key
+        self.base_url = base_url
+
+    def _headers(self):
+        return {
+            "X-API-Key": self.api_key,
+            "Content-Type": "application/json",
+        }
+
+    def _post(self, path, payload):
+        url = "{}{}".format(self.base_url, path)
+        resp = requests.post(url, json=payload, headers=self._headers())
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)